src/promptflow/promptflow/errors.py

```diff
@@ -0,0 +1,28 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+from promptflow.exceptions import UserErrorException, SystemErrorException, ValidationException
+
+
+class ValueErrorException(ValidationException, ValueError):
+    pass
+
+
+class NotImplementedErrorException(SystemErrorException, NotImplementedError):
```

Comment:
`Error` and `Exception` are a bit repetitive

Reply:
Yes, but I also referred to the existing writing method: **SystemErrorException**.